app/ui-react/syndesis/src/modules/integrations/components/IntegrationEditorStepAdder.tsx

```diff
@@ -131,13 +125,8 @@ function getWarningMessage(
     return (
       <>
         <Link
-          data-testid={
-            'integration-editor-step-adder-add-default-flow-link'
-          }
-          to={props.configureStepHref(
-            idx,
-            props.steps[idx]
-          )}
+          data-testid={'integration-editor-step-adder-add-default-flow-link'}
```

Comment:
There's kinda too many formatting only changes like this one, I think you need a `prettier` extension in your IDE to ensure these blocks get formatted properly.  Otherwise subsequent changes after this PR in these areas will contain extra noise.

Reply:
Or maybe you have it installed and someone else didn't in which case ignore me :-)

Reply:
That was pre-commit hooks doing, I think its using `prettier`.